gap/generic/KnownNilpotent.gi

```diff
@@ -60,21 +63,22 @@ RECOG.HomForNilpotent := function(data,el)
 end;
 
 RECOG.CalcNiceGensKnownNilpotent := function(ri,origgens)
-  local b;
-  b := List([1..Length(ri!.waytonice)],i->origgens[i]^ri!.waytonice[i][1]);
+  local kernelGens;
```

Comment:
```suggestion
  local kernelgens;
```
I used `kernelgens` elsewhere, because it fits better with `origgens`, nicegens`. We can rename them all to camelCase later on (or not, we'll have to see)